documentation/modules/auxiliary/scanner/winrm/winrm_auth_methods.md

```diff
@@ -0,0 +1,23 @@
+## Description
+This module sends a request to an HTTP/HTTPS service to see if it is a WinRM service. If it is a WinRM service, it also gathers the Authentication Methods supported.
+
```

Comment:
is this on by default, did you have to set anything to enable it to be vulnerable?